Formula/g/go.rb

```diff
@@ -1,9 +1,8 @@
 class Go < Formula
   desc "Open source programming language to build simple/reliable/efficient software"
   homepage "https://go.dev/"
-  url "https://go.dev/dl/go1.21.6.src.tar.gz"
-  mirror "https://fossies.org/linux/misc/go1.21.6.src.tar.gz"
```

Comment:
why do we remove the mirror?

Reply:
Oops, accidentially, because i did it with `brew bump-formula-pr` this time. Good catch! Will add it back.

Reply:
Done in 08591207b28c0ab640ebdee378499d3f40e1a025

Reply:
> Oops, accidentially, because i did it with brew bump-formula-pr this time. Good catch! Will add it back.

I have seen such issue before, using `brew bump --open-pr xxx` is a better solution.

My hunch is you directly specify `--url` which caused such issue.

Reply:
Correct, i used `--url=...`